include/itkLabelSetUtils.h

```diff
@@ -257,8 +257,9 @@ doOneDimensionErodeFirstPass(TInIter &          inputIterator,
   //  const RealType magnitude = (magnitudeSign * iscale * iscale)/(2.0 *
   // sigma);
   const RealType  magnitude = (magnitudeSign * iscale * iscale) / (2.0);
-  LineBufferType  lineBuf(lineLength);
-  LabelBufferType labBuf(lineLength);
+  const RealType  initValue = 0.0;
+  LineBufferType  lineBuf(lineLength, initValue);
+  LabelBufferType labBuf(lineLength, initValue);
```

Comment:
LabelBufferType is not of RealType, but of PixelType.

Reply:
:+1: Addressed in 663e0cb.